src/network/clientpackethandler.cpp

```diff
@@ -463,6 +463,11 @@ void Client::handleCommand_ActiveObjectRemoveAdd(NetworkPacket* pkt)
 		infostream << "handleCommand_ActiveObjectRemoveAdd: " << e.what()
 				<< ". The packet is unreliable, ignoring" << std::endl;
 	}
+
+	// m_activeobjects_received is false before the first
+	// TOCLIENT_ACTIVE_OBJECT_REMOVE_ADD packet is received
+	if (!m_activeobjects_received)
+		m_activeobjects_received = true;
```

Comment:
You could just set the variable without checking - the compiler probably does the same.

Reply:
I got a little greedy for performance and tried to reduce variable setting overhead by checking for the value first ;)

I'll change it though.